src/background/messaging/rpc-methods/sign-stacks-transaction.ts

```diff
@@ -37,21 +41,35 @@ import { trackRpcRequestError, trackRpcRequestSuccess } from '../rpc-message-han
 
 const MEMO_DESERIALIZATION_STUB = '\u0000';
 
-const cleanMemoString = (memo: string): string => {
+function cleanMemoString(memo: string): string {
   return memo.replaceAll(MEMO_DESERIALIZATION_STUB, '');
-};
+}
 
 function encodePostConditions(postConditions: PostCondition[]) {
   return postConditions.map(pc => bytesToHex(serializePostCondition(pc)));
 }
 
-const transactionPayloadToTransactionRequest = (
+function getStacksTransactionHexFromRequest(requestParams: StxSignTransactionRequestParams) {
+  if ('txHex' in requestParams) return requestParams.txHex;
+  return requestParams.transaction;
+}
+
+function getAccountAddressFromRequest(requestParams: StxSignTransactionRequestParams) {
+  if ('txHex' in requestParams) return requestParams.stxAddress;
```

Comment:
Just noting these appear copied, but not changed `txHex` here and below.

Reply:
that's intentional, this is the check to determine which payload type it is